examples/docs/es/date-picker.md

```diff
@@ -92,14 +92,15 @@
 </style>
 
 ## DatePicker
+Uso Date Picker para fecha entrada.
```

Comment:
No sería mejor, `Utiliza Date Picker para introducir fechas`?

Reply:
vale

Reply:
Hi @carmencitaqiu, could you update this PR to reflect this change?